client/src/components/Slider/Slider.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Slider from "./Slider";
+import Person from "../PersonCard/PersonCard";
+
+
+let cards = [<Person />, <Person />, <Person />, <Person />, <Person />];
+storiesOf("Atoms/Slider", module).add("Default", () => (
+    <Slider 
+    cards={cards}
+    
+    />
```

Comment:
I think a better way to do this would be to leverage `props.children` inside the `Slider` component. That would mean rendering it would instead look something like this:

```js
<Slider>
	<Person />
	<Person />
	<Person />
	<Person />
</Slider>
```